upgrade_test.py

```diff
@@ -268,6 +270,9 @@ def rollback_node(self, node):
         if self.params.get('test_sst3', default=None):
             node.remoter.run(
                 r'sudo sed -i -e "s/enable_sstables_mc_format:/#enable_sstables_mc_format:/g" /etc/scylla/scylla.yaml')
+        if self.params.get('test_upgrade_from_installed_3_1_0', default=None):
```

Comment:
Why do you need to check this parameter?
Why not just grepping the scylla.yaml for this param or just even "sed" it anyway...

Reply:
What I checked is parameter in SCT, not scylla.yaml.
We only to append `enable_3_1_0_compatibility_mode: true` for upgrade from 3.1.0, so it should be configurable.